org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/actions/SelectionUtil.java

```diff
@@ -266,19 +263,13 @@ public static MavenProject getMavenProject(IEditorInput editorInput, IProgressMo
       IPath path = storage.getFullPath();
       if(path == null || !new File(path.toOSString()).exists()) {
         File tempPomFile = null;
-        InputStream is = null;
-        OutputStream os = null;
-        try {
+        try (InputStream is = storage.getContents()) {
           tempPomFile = File.createTempFile("maven-pom", ".pom"); //$NON-NLS-1$ //$NON-NLS-2$
-          os = new FileOutputStream(tempPomFile);
-          is = storage.getContents();
-          IOUtil.copy(is, os);
+          Files.copy(storage.getContents(), tempPomFile.toPath());
```

Comment:
should this be
```suggestion
          Files.copy(is, tempPomFile.toPath());
```

Reply:
Yes you are right. Thanks for pointing out!